### PR TITLE
switch to different localstorage library

### PR DESCRIPTION
**Pull request by bluemaex**
to correctly support also safaris private mode

### CODE REVIEW

File: src/easybib-api-client.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ module.exports = function($injector, $q, $http, store, $timeout, easyBibApiAcces
       var deferred, accessData;
 
       deferred = $q.defer();
-      accessData = store.get('easybib-api-access-data');
+      accessData = self.store.get('easybib-api-access-data');
```
Comment (sashless) [inline src/easybib-api-client.js]: should be `ls.easybib-api-access-data' like everywhere else i think ?

Comment (sashless) [inline src/easybib-api-client.js]: well nope. You just changed the tests to use a different key ^^ Why ?

File: src/easybib-api-client_spec.js [hunk 2]
```diff
@@ -58,7 +58,7 @@ describe('EasyBib Api Client', function() {
 
   describe('get()', function() {
     it('should add access-token to request headers', function() {
-      localStorage.setItem('easybib-api-access-data',
+      localStorage.setItem('ls.easybib-api-access-data',
         JSON.stringify(fixtures.accessTokenResponse));
```
Comment (sashless) [inline src/easybib-api-client_spec.js]: why did you change that ?

Comment (bluemaex) [inline src/easybib-api-client_spec.js]: The new library prefixes everything to avoid collisions. See https://github.com/grevory/angular-local-storage 

That is also the reason why I just changed the key in the the tests when we manually use localstorage and not in the actual code :)

**Comment by sashless** [comment]
cool, lets merge :+1:
